tools/coverity_misra.config

```diff
@@ -32,7 +32,7 @@
         },
         {
             deviation: "Rule 5.4",
-            reason: "In C90 First 31 characters should be different; in C99 first 63 characters should be significant"
+            reason: "There a lot of macros which are present for backward compatibility in FreeRTOS+TCP. These macros have similar names without many unique characters which MISRA warns against."
```

Comment:
There a lot of macros -> There are a number of macros

Reply:
Done, nice catch.